README.md

```diff
@@ -7,12 +7,49 @@
-|------|-------------|------|---------|----------|
+| Name                        | Description                                                                                                          | Type           | Default                     | Required |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------- | -------------  | --------------------------- | :------: |
+| bucket_enable_encryption    | Set this to `true` to enable encryption on a created S3 bucket                                                       | `bool`         | `false`                     |    no    |
+| bucket_enable_mfa_delete    | Set this to `true` to require MFA for object deletion (Requires versioning)                                          | `bool`         | `false`                     |    no    |
+| bucket_enable_versioning    | Set this to `true` to enable access versioning on a created S3 bucket                                                | `bool`         | `false`                     |    no    |
+| bucket_force_destroy        | Force destroy bucket (Required when bucket not empty)                                                                | `bool`         | `false`                     |    no    |
+| cloudwatch_region           | The region, to stream CloudWatch logs from                                                                           | `string`       |                             |    yes   |
+| cluster_names               | A list of cluster names, within the cloudwatch region to integrate with                                              | `list(string)` |                             |    yes   |
+| external_id_length          | The length of the external ID to generate. Max length is 1224.                                                       | `number`       | `16`                        |    no    |
+| integration_name            | The name of the AWS EKS Audit Log integration in Lacework                                                            | `string`       | `TF AWS EKS Audit Log`      |    no    |
+| lacework_aws_account_id     | The Lacework AWS account that the IAM role will grant access                                                         | `string`       | `"434813966438"`            |    no    |
+| prefix                      | The prefix that will be use at the beginning of every generated resource                                             | `string`       | `"lacework-s3-data-export"` |    no    |
+| tags                        | A map/dictionary of Tags to be assigned to created resources                                                         | `map(string)`  | `{}`                        |    no    |
+| wait_time                   | Amount of time to wait before the next resource is provisioned.                                                      | `string`       | `"10s"`                     |    no    |
```

Comment:
This could perhaps be described more specifically as the time to wait between setting up AWS resources, and creating the Lacework integration?
If there's an undetermined amount of time needed by AWS it might be helpful to wait a more excessive amount of time to reduce the risk of users seeing an issue. 10s feels short.

Reply:
Happy to update the wording of this. The default value is the same as what we use in other AWS modules. I'm not sure that we need to make it any longer as default.